### PR TITLE
add mention to Bark in TTS chapter

### PR DESCRIPTION
**Pull request by ylacombe**
Hi @MKhalusova,

Bark is a TTS model that was recently [added to transformers](https://huggingface.co/docs/transformers/model_doc/bark). As per @sanchit-gandhi request, I've added a Bark mention in the TTS chapter!
I would love to have your feedback on it.

Many thanks,
Yoach

### CODE REVIEW

File: chapters/en/chapter6/pre-trained_models.mdx [hunk 1]
```diff
@@ -183,6 +183,94 @@ pre-trained checkpoint only supports English language:
 	height="450">
 </iframe>
 
+## Bark
+
+Bark is a transformer-based text-to-speech model proposed by Suno AI in [suno-ai/bark](https://github.com/suno-ai/bark).
+
+Bark is made of 4 main models:
```
Comment (sanchit-gandhi) [inline chapters/en/chapter6/pre-trained_models.mdx:190]: A diagram would be nice here! Especially for course material. No strong need to do it now, but in the future it's worth considering when the architecture gets more complicated (see https://huggingface.co/learn/audio-course/chapter7/voice-assistant)

File: chapters/en/chapter6/pre-trained_models.mdx [hunk 2]
```diff
@@ -183,6 +183,94 @@ pre-trained checkpoint only supports English language:
...
+
+```python
+from transformers import BarkModel, BarkProcessor
+
+model = BarkModel.from_pretrained("suno/bark-small")
+processor = BarkProcessor.from_pretrained("suno/bark-small")
+```
+
+
+<Tip>
```
Comment (sanchit-gandhi) [inline chapters/en/chapter6/pre-trained_models.mdx:215]: I wouldn't write these all as tips, but rather give one example for each (as you did for the first Bark Colab you released) -> this makes it easier for the reader to understand how each tip relates to the code example

File: chapters/en/chapter6/pre-trained_models.mdx [hunk 3]
```diff
@@ -196,7 +284,7 @@ VITS is a speech generation network that converts text into raw speech waveforms
 auto-encoder, estimating audio features from the input text. First, acoustic features, represented as spectrograms, are 
 generated. The waveform is then decoded using transposed convolutional layers adapted from HiFi-GAN. 
 During inference, the text encodings are upsampled and transformed into waveforms using the flow module and HiFi-GAN decoder.
-This means that you do not need to add a vocoder for inference, it's already "built-in".  
+Like Bark, there's no need for a vocoder, as waveforms are generated directly.
```
Comment (sanchit-gandhi) [inline chapters/en/chapter6/pre-trained_models.mdx:287]: Nice!

File: chapters/en/chapter6/pre-trained_models.mdx [hunk 4]
```diff
@@ -183,6 +183,94 @@ pre-trained checkpoint only supports English language:
...
+Second one:
+```python
+Audio(speech_output[1], rate=sampling_rate)
+```
+
+Third one:
+```python
+Audio(speech_output[2], rate=sampling_rate)
+```
+
```
Comment (sanchit-gandhi) [inline chapters/en/chapter6/pre-trained_models.mdx:273]: Feel free to link your notebook on optimising Bark here too!

**Review by sanchit-gandhi — COMMENTED** [review]
Think we can split the `Tip` into standalone code examples, each prepended with a written explanation of what we're doing, much like you did for the first Bark Colab! Otherwise LGTM

**Comment by sanchit-gandhi** [comment]
Once this PR is merged we can also announce the course update in the audio discord channel to try and boost engagement a bit (cc @Vaibhavs10 @MKhalusova)

File: chapters/en/chapter6/pre-trained_models.mdx [hunk 5]
```diff
@@ -183,6 +183,94 @@ pre-trained checkpoint only supports English language:
...
+from transformers import BarkModel, BarkProcessor
+
+model = BarkModel.from_pretrained("suno/bark-small")
+processor = BarkProcessor.from_pretrained("suno/bark-small")
+```
+
+
+<Tip>
+
+Bark is very versatile and can generate audio conditioned by [a speaker embeddings library](https://suno-ai.notion.site/8b8e8749ed514b0cbf3f699013548683?v=bc67cff786b04b50b3ceb756fd05f68c), which you can listen to here, and which can be loaded via the processor.
```
Comment (MKhalusova) [inline chapters/en/chapter6/pre-trained_models.mdx:217]: "which you can listen to here, " - where? was there supposed to be a link?

**Review by MKhalusova — APPROVED** [review]
Thanks for adding a section about Bark! Overall, it looks good to me, I've only left a minor nit.

**Comment by ylacombe** [comment]
Hi @sanchit-gandhi and @MKhalusova , I've corrected the text according to your comments! Many thanks for the review!

File: chapters/en/chapter6/pre-trained_models.mdx [hunk 6]
```diff
@@ -183,6 +183,124 @@ pre-trained checkpoint only supports English language:
...
+```
+
+
+Bark is very versatile and can generate audio conditioned by [a speaker embeddings library](https://suno-ai.notion.site/8b8e8749ed514b0cbf3f699013548683?v=bc67cff786b04b50b3ceb756fd05f68c) which can be loaded via the processor.
+
+```python
+# add a speaker embedding
+inputs = processor("This is a test!", voice_preset="v2/en_speaker_3")
+
+speech_output = model.generate(**inputs).cpu().numpy()
```
[the remark this answers is not in the document]
Comment (MKhalusova) [reply chapters/en/chapter6/pre-trained_models.mdx:221]: CC @ylacombe. Can you please address this feedback so we could merge the PR?

**Comment by sanchit-gandhi** [comment]
Still think this is worth merging when you get the chance @ylacombe

File: chapters/en/chapter6/pre-trained_models.mdx (hunk 6 above)
[the remark this answers is not in the document]
Comment (MKhalusova) [reply chapters/en/chapter6/pre-trained_models.mdx:221]: If it's fine with @sanchit-gandhi , I'll merge without the embedded audio examples.

**Comment by ylacombe** [comment]
Hey @sanchit-gandhi  and @MKhalusova , sorry for the delay! I've added audio samples as well!

File: chapters/en/chapter6/pre-trained_models.mdx [hunk 7]
```diff
@@ -221,6 +221,11 @@ inputs = processor("This is a test!", voice_preset="v2/en_speaker_3")
 speech_output = model.generate(**inputs).cpu().numpy()
 ```
 
+<audio controls> 
+  <source src="https://huggingface.co/datasets/ylacombe/hf-course-audio-files/resolve/main/first_sample.wav" type="audio/wav">
```
[the remark this answers is not in the document]
Comment (ylacombe) [reply chapters/en/chapter6/pre-trained_models.mdx:225]: Done !

**Pull request merged**